pulsar/impl_message_test.go

```diff
@@ -80,6 +80,12 @@ func TestAckTracker(t *testing.T) {
 
 	}
 	assert.Equal(t, true, tracker.completed())
+
+	// test large number 1000 cumulative
+	tracker = newAckTracker(1000)
+
+	assert.Equal(t, true, tracker.ackCumulative(999))
```

Comment:
It's better to add a test to verify `ackCumulative` for 0 to 998 returns false.

```golang
        tracker = newAckTracker(1000)
        for i := 0; i < 999; i++ {
                assert.False(t, tracker.ackCumulative(i))
        }
        assert.False(t, tracker.completed())
```

Reply:
Good suggestion! I will add test code to cover the cumulative tracker.